dist/js/joomla-a11y-checker.umd.js

```diff
@@ -1300,40 +1309,41 @@
                 <span class='sa11y-outline-list-item sa11y-yellow-text sa11y-bold'>${htext}</span>
             </li>`;
 
-                  if ($el.not(this.options.outlineIgnore).length !== 0) {
+              let ignoreArray = [];
+              if (sa11yOutlineIgnore) {
```

Comment:
such params in our code will be: `this.options.outlineIgnore`

Reply:
yes I literally just checked that one